extractors/numbers/ip_extraction/config.py

```diff
@@ -15,8 +15,12 @@ def get_config():
         available_for=["refinery", "common"],
         part_of_group=[
             "numbers",
+            "personal_identifiers"
         ],  # first entry should be parent directory
         # bricks integrator information   
+        cognition_init_mapping={
+            "@@LABEL@@": "IP-address"
```

Comment:
to my knowledge this label wasn't planned for references PII task. So i would assume that this birck isn't part of the group & doesn't get a mapping. Alternativly i might not have the newest information regarding the wizard and we want to add IP-Address.

Reply:
from notion `IP-address`

Reply:
@LeonardPuettmannKern What is the current status regarding this comment?

Reply:
Planned for references and queries.
@LeonardPuettmannKern We only need to change the label to `IP address` to match the notion page

Reply:
Should be conform with the Notion labels now.